src/index.js

```diff
@@ -0,0 +1,18 @@
+// import _ from 'lodash';
+
+// function component() {
+//     const element = document.createElement('div');
+
+//     element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
+//     return element;
+// }
+
+// document.body.appendChild(component());
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const temp = <h1>Hello there!</h1>; // variant to check babel
+// const temp = React.createElement('h1', {className: 'title'}, 'Hello there!'); //right variant
```

Comment:
In the future create react elements only with JSX

Reply:
ok